readme.md

```diff
@@ -16,19 +16,30 @@ $ source .venv/bin/activate
 (.venv) $ pip3 install -r requirements.txt
 ```
 
-Note: To step out of the virtual environment, run `$ deactivate`.
+If you will finish working in virtual environment you can step out of it:
 
-## Build it
+```console
+(.venv) $ deactivate
+```
 
+## Build it
+To build project you need to be in virtual environment.
```

Comment:
s/build/build the/

and

s/in virtual environment/in the virtual environment/

Reply:
Fixed